ragna/core/_components.py

```diff
@@ -76,6 +92,22 @@ def _protocol_model(cls) -> Type[pydantic.BaseModel]:
         return merge_models(cls.display_name(), *cls._protocol_models().values())
 
 
+@dataclass
+class Embedding:
+    embedding: list[float]
```

Comment:
Should we call this

```suggestion
    values: list[float]
```

`embedding.embedding` always feels wrong.